aksel.nav.no/website/pages/eksempler/skeleton/link-card.tsx

```diff
@@ -0,0 +1,59 @@
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import { Box, HStack, Skeleton } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <Box
+      paddingBlock="space-16"
+      paddingInline="space-20"
+      borderRadius="xlarge"
+      borderWidth="1"
+      style={{
+        border:
+          "1px solid var(--a-border-subtle, var(--ax-border-neutral-subtleA))",
+      }}
```

Comment:
Can we just use the borderColor-prop?

```suggestion
      borderColor="border-subtle"
```

Reply:
Example won't work for legacy and darkside then 💥